### PR TITLE
Add support for tiny PEFT-based Flux LoRA based on TheLastBen's post on Reddit

### PR DESCRIPTION
**Pull request by mhirki**
As suggested on Reddit by TheLastBen:
https://www.reddit.com/r/StableDiffusion/comments/1f523bd/good_flux_loras_can_be_less_than_45mb_128_dim/

I'm currently testing --flux_lora_target=tinier with a rank 16 LoRA and it's having some effect:

| 100 steps | 1300 steps |
| --- | --- |
| step_100_validation_1024x1024 | step_1300_validation_1024x1024 |

File size is only 576 kB for this ultra-tiny LoRA. My learning rate is at 1e-3 with adamw_bf16 optimizer.

### CODE REVIEW

File: helpers/training/adapter.py [hunk 1]
```diff
@@ -77,6 +77,19 @@ def determine_adapter_target_modules(args, unet, transformer):
...
                 "proj_out",
             ]
+        elif args.flux_lora_target == "tiny":
+            # From TheLastBen
+            # https://www.reddit.com/r/StableDiffusion/comments/1f523bd/good_flux_loras_can_be_less_than_45mb_128_dim/
+            target_modules = [
+                "single_transformer_blocks.7.proj_out",
+                "single_transformer_blocks.20.proj_out",
+            ]
+        elif args.flux_lora_target == "tinier":
```
Comment (bghira) [inline helpers/training/adapter.py]:
```suggestion
        elif args.flux_lora_target == "nano":
```

**Comment by mhirki** [comment]
Damnit, you merged it too fast, I still needed to change "tinier" to "nano" in other places

**Comment by bghira** [comment]
it wasn't marked as draft 😓 no worries. i was going to add a note to the flux quickstart anyway.
